rust/feature-flags/src/flag_definitions.rs

```diff
@@ -74,15 +71,15 @@ pub struct MultivariateFlagOptions {
 pub struct FlagFilters {
     pub groups: Vec<FlagGroupType>,
     pub multivariate: Option<MultivariateFlagOptions>,
-    pub aggregation_group_type_index: Option<u8>,
+    pub aggregation_group_type_index: Option<i8>,
     pub payloads: Option<serde_json::Value>,
     pub super_groups: Option<Vec<FlagGroupType>>,
 }
 
 #[derive(Debug, Clone, Deserialize)]
 pub struct FeatureFlag {
-    pub id: i64,
-    pub team_id: i64,
+    pub id: i32,
```

Comment:
switched types here to reflect db type, otherwise it'd complain about the conversion